system-tests/fixtures/ios-container/ElectrodeContainer/ElectrodeReactNative.m

```diff
@@ -128,13 +129,24 @@ @implementation ElectrodeReactNative
 #pragma mark - Public Methods
 
 + (void)startWithConfigurations:(id<ElectrodePluginConfig>)reactContainerConfig
+{
+    id sharedInstance = [ElectrodeReactNative sharedInstance];
+    static dispatch_once_t startOnceToken;
+    semaphore = dispatch_semaphore_create(0);
+    dispatch_once(&startOnceToken, ^{
+        [sharedInstance startContainerWithConfiguration:reactContainerConfig ernDelegate:nil
+];
```

Comment:
Not sure about the formatting on this line, can you double check and confirm @jw2175

Reply:
Yeah, it's not pretty but works. I believe the previous version has similar formatting when there are no configs set. Not too familiar with the code generation portion to make the brackets on the same line.